tutorial.py

```diff
@@ -8,11 +8,15 @@ def sing_song():
     j = 30
     k = i + j
     print(k)
-
+    
+def lovely_changes():
+    print("Excited to see its result")
+    
 def print_song(input_is):
     print("What is a input {}".format(input_is))
 
```

Comment:
It is not necessary as it is a cosmetic feature so we can kind of take it ahead without further changes and hence thanks for the suggestion

Reply:
Thank you for your reply i ll be merging this code asap.